### PR TITLE
Multilingual support

### LINKED ISSUE #44: Add support for translations
The static engine behind the website, hugo, supports [translations](https://gohugo.io/content-management/multilingual/).

There's just a few tasks to get started:

- [ ] translate one piece of content
- [ ] add a i18n partial to list available languages for a given article, e.g. in a drop-down menu
- [ ] add translations for some base parameters like languages etc. maybe?

If anyone wants to translate a tutorial, send me a message and I'll work on this.

### PR DESCRIPTION
**Pull request by NathanLovato**
Closes #44

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -150,15 +113,111 @@ unsafe = true
...
 
 [languages.ja]
 languageName = "日本語"
 weight = 4
+[[languages.ja.menu.main]]
+name        = "チュートリアル"
+url         = "/tutorial/"
+weight      = 1
+[[languages.ja.menu.main]]
+name        = "道具"
```
Comment (jtakakura) [inline config.toml:205]: I think it's better to translate as "ツール" as it is.

Comment (NathanLovato) [reply config.toml:205]: Thanks! 🙂

Comment (NathanLovato) [reply config.toml:205]: Are the other menu entries fine? In particular, I wasn't sure about 製品 for products (we mostly have courses but also the Krita brushes).

Comment (jtakakura) [reply config.toml]: It seems it is a little bit of formal expression, I think.
It is described "製品" is "プロダクト" on many websites, especially tech company websites.

Comment (NathanLovato) [reply config.toml]: Fixed, thanks!

**Pull request merged**